src/data/balance/pokemon-evolutions.ts

```diff
@@ -77,6 +77,7 @@ export enum EvolutionItem {
   LEADERS_CREST
 }
 
+const tyrogueMoves = [MoveId.LOW_SWEEP, MoveId.MACH_PUNCH, MoveId.RAPID_SPIN];
```

Comment:
```suggestion
const tyrogueMoves = [MoveId.LOW_SWEEP, MoveId.MACH_PUNCH, MoveId.RAPID_SPIN] as const;
```

probably should make this readonly